src/extension/src/constants.ts

```diff
@@ -260,6 +260,16 @@ export const CONSTANTS = {
     MAX_LENGTH: 60,
     MIN_LENGTH: 3
   },
+  ASP_NAME: {
```

Comment:
Easily confused with ASP.NET :P

Reply:
do you suggest to change the constant name ?